ui/src/frontend/help_modal.ts

```diff
@@ -168,6 +168,9 @@ class KeyMappingsHelp implements m.ClassComponent {
             m('tr',
               m('td', keycap(ctrlOrCmd), ' + ', keycap('a')),
               m('td', 'Select all')),
+            m('tr',
+              m('td', keycap('v')),
+              m('td', 'Show vsync points in columns across all tracks')),
```

Comment:
I think AGi's shorter description is intuitive enough. I'd go with that: `Highlight VSyncs`. But @csteinberg-samsung should decide :-)

Reply:
I agree, I think Highlight VSyncs is a good command title, if there were a description field or tool tip I think this would have been a good description.